editions/2015-02.md

```diff
@@ -0,0 +1,61 @@
+
+
+### [Cuttlefish is now an official Poplus component](http://poplus.org/posts/cuttlefish/)
+
+Cuttlefish is an easy-to-set-up transactional email server developed by OpenAustralia Foundation Founder [Matthew Landauer](https://twitter.com/matthewlandauer). Last year the OpenAustralia Foundation was awarded a Poplus mini grant to create [a hosted version of Cuttlefish](https://cuttlefish.oaf.org.au/) which is freely available to the Poplus and international civic tech communities. Read Matthew’s post, *[Email is your secret weapon](http://poplus.org/posts/email-is-your-secret-weapon/)*, on the Poplus blog to find out more. Now Cuttlefish has been accepted as an official component, more people will find out about and use it.
+
+[Poplus](http://poplus.org/) is a global federation for civic tech, developing components that can support projects everywhere. Join the [Poplus maillist](http://poplus.org/get-involved/) for updates and discussion.
+
+
+### [Federal Government announces the Digital Transformation Office](https://www.pm.gov.au/media/2015-01-23/establishment-digital-transformation-office)
+
+The Government is establishing a “small team of developers, designers, researchers and content specialists working across government ... to make services simpler, clearer and faster for Australian families and businesses.” This is an initiative that the Australian civic tech community have been looking forward to since the establishment of the [Government Digital Services (GDS) office in the UK](https://www.gov.uk/). It is an important project for the civic tech community to get involved with and help make it as effective as possible. You can read more about the DTO at [the Communication Minister’s FAQ](http://www.malcolmturnbull.com.au/policy-faqs/faqs-the-digital-transformation-office). GDS have also written [a blog post](https://gds.blog.gov.uk/2015/01/26/simpler-clearer-australia/) about the project.
+
+
+### [Growstuff](http://www.growstuff.org/)
+
+Growstuff is ‘a community of food gardeners who are building an open source platform to help you learn about growing food, track what you plant and harvest, and swap seeds and produce with other gardeners near you.’
+
+We love this one because it’s a practical open source project helping people grow food and community. It helps keep information held in plants and seeds open by making it easy to share. Growstuff releases the data it collects under open licenses, giving power to it’s developers as well as growing a community of gardeners. It promotes pair programming as the norm.
+
+This is a very active project. Get involved as a volunteer to help test, write for [the Growstuff blog](http://blog.growstuff.org/), [contribute code to the application](https://github.com/growstuff) or just dig in, grow some food and share your seeds and data.
```

Comment:
Fixed in https://github.com/openaustralia/newsletter/commit/394536548e1f5e15e67b48a3e155c777adfacb8d